Doc/library/dataclasses.rst

```diff
@@ -491,6 +491,27 @@ depend on one or more other fields.  For example::
         def __post_init__(self):
             self.c = self.a + self.b
 
+The :meth:`__init__` method generated by :func:`dataclass` does not call base
+class :meth:`__init__` methods. If the base class has an :meth:`__init__` method
+that has to be called, it is common to call this method in a
+:meth:`__post_init__` method::
+
+    @dataclass
+    class Rectangle:
+        height: float
+        width: float
+
+    @dataclass
+    class Square(Rectangle):
+        side: float
+
+        def __post_init__(self):
+            super().__init__(self.side, self.side)
+
+Note, however, that in general the dataclass-generated :meth:`__init__` methods
+not need to be called, since the derived dataclass will take care of
```

Comment:
"not need to" isn't correct. Apologies if that was my error. Maybe simpler language is needed, like "... methods don't need to be called, since ...`.

Reply:
Sorry, English is not my native tongue